docs/sdk/migrations/3.0.0/overview.md

```diff
@@ -14,16 +15,18 @@ The **Lit JS SDK V3** replaces the existing access control condition based encry
 
 <StateOfTheNetwork/>
 
+<OldMigrationNotice/>
+
 ## Data Migration
 
 
 :::caution
 
-The Lit development team will not be migrating the access control conditions that have been "stored" in the `jalapeno` and `serrano` networks but we will continue to maintain support for them for a finite period of time. Therefore, we will request users to migrate to `habanero` and `manzano` networks.
+The Lit development team will not be migrating the access control conditions that have been "stored" in the `jalapeno` and `serrano` networks but we will continue to maintain support for them for a finite period of time. Therefore, we will request users to migrate to `Datil` and `Datil-test` networks.
 
 :::
 
-If you wish to continue using these access control conditions for encryption or signing JWTs, please continue using the V2 SDK. But we strongly recommend users to migrate to V3 SDK as it supports our new networks, `habanero` and `manzano`, which are not supported by V2 SDK.
+If you wish to continue using these access control conditions for encryption or signing JWTs, please continue using the V2 SDK. But we strongly recommend users to migrate to V3 SDK as it supports our new networks, `Datil` and `Datil-test`, which are not supported by V2 SDK.
```

Comment:
Update v3 to v6.4 and above

Reply:
Updated this. The new networks on the old migration guides (2->3, 3->3.2, etc.) can get confusing